kpet/cmd_run.py

```diff
@@ -143,7 +143,7 @@ def main_create_baserun(args, database):
         src_set = patch.get_src_set_from_location_set(set(args.mboxes),
                                                       cookies)
     else:
-        src_set = None
+        src_set = {'all'}
```

Comment:
While this line fixed the tests I think this needs to be thought about as it could introduce other issues